src/test/java/org/janelia/saalfeldlab/n5/zarr/N5ZarrTest.java

```diff
@@ -144,6 +144,29 @@ public void testCreateNestedDataset() {
 		}
 	}
 
+	@Test
+	public void testCreateDatasetNameNull() {
```

Comment:
Can you please change this test to be called `testCreateDatasetNameEmpty`

Reply:
done